src/services/ui/src/pages/detail/index.tsx

```diff
@@ -45,21 +41,37 @@ const DetailCardWrapper = ({
 const StatusCard = ({
   status,
   raiWithdrawEnabled,
+  raiRecievedDate,
 }: {
   status: string;
   raiWithdrawEnabled: boolean;
+  raiRecievedDate: string;
 }) => {
-  const transformedStatuses = getStatus(SEATOOL_STATUS.WITHDRAWN);
+  const transformedStatuses = getStatus(status);
+  const { data: user } = useGetUser();
+
   return (
     <DetailCardWrapper title={"Status"}>
       <div>
-        <h2 className="text-xl font-semibold">{status}</h2>
-        {raiWithdrawEnabled &&
-        !Object.values(transformedStatuses).includes(status) ? (
+        <h2 className="text-xl font-semibold">
+          {user?.isCms &&
+          !user.user?.["custom:cms-roles"].includes(UserRoles.HELPDESK)
+            ? transformedStatuses.cmsStatus
+            : transformedStatuses.stateStatus}
+        </h2>
+        {raiWithdrawEnabled && (
           <em className="text-xs">
-            {"Withdraw Formal RAI Response - Enabled"}
+            {"Withdraw Formal RAI Response - Enabled"}{" "}
```

Comment:
The extra space can be removed at the end of this line

Reply:
@kevinhaube The space is needed because without it there is no space between the 2nd clock and the withdraw formal RAI is enabled status when it is displayed. this was a bug that Padma found.